src-backend/api/tests/test_model_element.py

```diff
@@ -0,0 +1,109 @@
+from django.test import TestCase
+from django.db import IntegrityError
+from nose.tools import raises, assert_equals, assert_not_equals
+from api.models import Element
+from utils import factories
+
+
+class ElementTest(TestCase):
+    def test_elements_required(self):
+        page = factories.PageFactory()
+        element = Element.objects.create(
+            display_index=0,
+            page=page
+        )
+
+        assert_equals(element.display_index, 0)
+        assert_equals(element.page, page)
+
+    def test_elements_all_properties(self):
+        page = factories.PageFactory()
+        Element.objects.create(
+            display_index=0,
+            eid='eid',
+            element_type='SELECT',
+            choices='[one]',
+            numeric='DIALPAD',
+            concept='TEST',
+            question='test question',
+            answer='',
+            page=page
+        )
+
+        element = Element.objects.get(concept='TEST')
+
+        assert_equals(element.display_index, 0)
+        assert_equals(element.eid, 'eid')
+        assert_equals(element.element_type, 'SELECT')
+        assert_equals(element.choices, '[one]')
```

Comment:
it's just a test convention

at the very least, we don't have to worry about some test failing down the line because the setup had a typo